crates/bitwarden-vault/src/cipher/cipher_client/mod.rs

```diff
@@ -184,6 +187,13 @@ impl CiphersClient {
         let decrypted_key = cipher_view.decrypt_fido2_private_key(&mut key_store.context())?;
         Ok(decrypted_key)
     }
+
+    /// Returns a new client for performing admin operations.
```

Comment:
🎨 Since this is most discoverable place for the `CipherAdminClient`, it might be beneficial to reiterate that these operations use special admin endpoints and do not modify local state. Similar to what you have on the `CipherAdminClient` documentation itself.
```suggestion
    /// Returns a new client for performing admin operations.
    /// Uses the admin server API endpoints and does not modify local state.
```